src/app/components/profile/profile.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { ShowUser } from 'src/app/models/show-user';
+import { AuthService } from 'src/app/services/auth.service';
+
+@Component({
+  selector: 'app-profile',
+  templateUrl: './profile.component.html',
+  styleUrls: ['./profile.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7182177)
```suggestion
  styleUrls: ["./profile.component.css"]
```